docker/dockerfiles/bedrock_functional_tests

```diff
@@ -0,0 +1,28 @@
+FROM python:2
+
+WORKDIR /app
+
+# Defaults
+ENV PYTEST_PROCESSES 5
+ENV BASE_URL https://www-dev.allizom.org
+ENV DRIVER Remote
+ENV SELENIUM_PORT 4444
+ENV BROWSER_NAME firefox
+ENV PRIVACY "public restricted"
+ENV TESTS_PATH /app/tests/functional
+ENV RESULTS_PATH /app/results
+ENV PYTHONDONTWRITEBYTECODE=1
+
+COPY ./requirements /app/requirements
+# Pin a known to work with peep pip version.
+ENV PIP_DISABLE_PIP_VERSION_CHECK=1
+RUN pip install -r requirements/pip.txt
+
+# Install requirements
+COPY ./bin/peep.py /app/bin/peep.py
+RUN ./bin/peep.py install --no-cache-dir -r requirements/test.txt
+
+COPY . /app
+
+# Run the tests
+CMD ["/app/bin/run-functional-tests.sh"]
```

Comment:
This is fine here. For an extra optimization step it can be moved after `WORKDIR`. It only defines the command to run on `docker run` and it doesn't have to be on the end of the file.